src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorPagesWebSite.SimpleForms.html

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
This may actually cause a new failure because response content is trimmed before comparing with the baseline. I'm testing locally to confirm. If that's true, I'll update the `GENERATE_BASELINES` part of the test to stabilize things.